modules/apps/archived/frontend-editor-tinymce-web/src/main/resources/META-INF/resources/tinymce.jsp

```diff
@@ -143,27 +143,25 @@ name = HtmlUtil.escapeJS(name);
 			var url = tinymce.activeEditor.settings[browseUrls[meta.filetype]];
 
 			if (url) {
-				var openItemSelectorDialog = function (itemSelectorDialog) {
-					itemSelectorDialog.eventName = '<%= name %>selectItem';
-					itemSelectorDialog.singleSelect = true;
-					itemSelectorDialog.url = url;
+				let zIndex;
 
-					var tinymceDialogContainer = document.querySelector(
-						'.tox-tinymce-aux'
-					);
+				var tinymceDialogContainer = document.querySelector(
```

Comment:
`let` and `var` in a JSP. Is that legal?

Reply:
ah, right. I may have forgotten to run our formatter...